addons/reviewstack/loader.mjs

```diff
@@ -0,0 +1,21 @@
+// Found here: https://stackoverflow.com/a/74931879
+import {existsSync} from 'fs';
```

Comment:
Hi Suyog, good to hear from you!

Would you mind running this through Prettier or something so we terminate lines with `;` where appropriate, prefer `const` to `let`, etc.?